docs/website/docs/overview/installation.md

```diff
@@ -298,6 +298,13 @@ type odo.exe.sha256
    odo version
    ```
 
+### Installing `odo` using Maven plugin
+It is possible to integrate the odo binary download in a Maven project using [odo Downloader Plugin](https://github.com/tnb-software/odo-downloader).
+The download can be executed using `download` goal which automatically retrieves the version for the current architecture:
+```shell
+mvn software.tnb:odo-downloader-maven-plugin:0.1.3:download -Dodo.target.file=$HOME/bin/odo
```

Comment:
Thanks for your contribution, @mcarlett !

Since this is the documentation for odo v3 (currently not GA'ed), I think this command should also pin the version of odo to install (v3.0.0-alpha2, to be consistent with the version used in the other installation methods in this file).
I see that this is possible with the `odo.version` property.

```suggestion
mvn software.tnb:odo-downloader-maven-plugin:0.1.3:download \
  -Dodo.target.file=$HOME/bin/odo \
  -Dodo.version=v3.0.0~alpha2
```

Otherwise, this would download the "latest" version, which would point to the latest stable version (2.5.1 at this time).
(Well, this won't be true if you test it right now, because we are currently releasing odo v3-beta1, and the latest link temporarily points to that version, but that should be fixed very soon).

We will then keep this `odo.version` property in sync.

Reply:
thanks @rm3l for the tip about the version, you're absolutely right! excuse me for missing the correct version

Reply:
> thanks @rm3l for the tip about the version, you're absolutely right! excuse me for missing the correct version
> 
No worries - thank you for the changes.

> we need to be updated after https://github.com/redhat-developer/odo/pull/5919
>
Yes, exactly. I've added a note in #5919